src/sql/platform/connection/common/interfaces.ts

```diff
@@ -18,3 +18,22 @@ export interface IConnectionProfileStore {
 	id: string;
 }
 
+export enum ServiceOptionType {
```

Comment:
Copied these from the sqlexthost types. Could probably remove those types eventually.